package.json

```diff
@@ -45,8 +46,12 @@
     "cssnano": "4.1.10",
     "dotenv": "8.1.0",
     "dotenv-webpack": "1.7.0",
-    "eslint": "6.4.0",
-    "eslint-config-edx": "4.0.4",
+    "eslint": "6.8.0",
```

Comment:
eslint-config-airbnb defines it's dependencies as peer dependencies - our @edx/eslint-config now does the same.  That means frontend-build - as the user of our config - is responsible for installing the peer deps.  An MFE that used @edx/eslint-config directly would have to do the same thing, which is the behavior of eslint-config-airbnb too.

Reply:
It's a bit odd, as eslint-config-airbnb expects them to be installed as dev dependencies, which would normally make sense.  In the case of frontend-build, it _feels_ as if they should be installed as full dependencies since this is a build-time tool.  We may eventually iterate on this...

Reply:
I think you're right to make them full dependencies. Otherwise they wouldn't be installed when frontend-build is added as a dev dependency of an MFE.